CSLabs.Api/Startup.cs

```diff
@@ -59,25 +61,36 @@ public void ConfigureServices(IServiceCollection services)
                     options.SerializerSettings.NullValueHandling = NullValueHandling.Ignore;
                 });
 
-            ConfigureEmail(services, appSettings.Email);
+            ConfigureEmail(services, appSettings);
             services.ConfigureDatabase(appSettings.ConnectionStrings.DefaultConnection);
             ConfigureCors(services, appSettings.CorsUrls);
             ConfigureJWT(services, appSettings.JWTSecret);
             services.AddAutoMapper(Assembly.GetExecutingAssembly());
             services.ProvideAppServices();
         }
 
-        private void ConfigureEmail(IServiceCollection services, EmailSettings emailSettings)
+        private void ConfigureEmail(IServiceCollection services, AppSettings appSettings)
         {
             Console.WriteLine(Environment.CurrentDirectory);
-            if (string.IsNullOrEmpty(emailSettings.FromAddress))
+            if (string.IsNullOrEmpty(appSettings.AWS.SES.FromAddress))
             {
                 throw new ConfigurationException("Email.FromAddress must be configured in the appsettings.json. Please follow the setup steps in the readme.");
             }
-            services
-                .AddFluentEmail(emailSettings.FromAddress)
-                .AddRazorRenderer(Path.Join(Environment.CurrentDirectory, "Views"))
-                .AddSmtpSender(emailSettings.Host, emailSettings.Port, emailSettings.UserName, emailSettings.Password);
+
+            if (!appSettings.Deployment.Equals("production"))
+            {
+                services
+                    .AddFluentEmail(appSettings.AWS.SES.FromAddress)
+                    .AddRazorRenderer(Path.Join(Environment.CurrentDirectory, "Views"))
+                    .AddSESSender(appSettings.AWS.AccessKey, appSettings.AWS.SecretKey, RegionEndpoint.USEast2);
+            }
+            else
+            {
+                services
+                    .AddFluentEmail(appSettings.AWS.SES.FromAddress)
```

Comment:
You should add "-dev" to the dev email. We want to delineate between the environments.

Reply:
Let's do this in the configuration